MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1150 @@
+        {
+            dateItem.MinimumDate = new DateTime(2022, 5, 5);
+            dateItem.MaximumDate = new DateTime(2022, 9, 2);
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Time editor
+
+In the time editor, the [TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows) will be loaded.
+
+In the time editor, the default time value (12:00 AM) is displayed.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Time)]
+    [Display(Name = "Event Time")]
+    public TimeSpan? EventTime { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### Change the format of time editor**
+
+In the time editor, short time will be shown by default. You can change the applied format by setting the `Format`property in `DataFormTimeItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "EventTime" && e.DataFormItem is DataFormTimeItem timeItem)
+        {
+            timeItem.Format = "HH:mm";
+        }
+    } 
+
+{% endhighlight %}
+{% endtabs %}
+
+## CheckBox editor
+
+In [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) control is loaded. By default, for `bool` data type property, the [CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0) editor will be loaded in data form.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing the checkbox color
+
+By default the checkbox color is blue, you can change the checkbox color by using `Color` property in `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+    
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem.FieldName == "agree" && e.DataFormItem is DataFormCheckBoxItem dataFormCheckBox)
+        {
+            dataFormCheckBox.Color = Colors.Pink;
+        }
+    }
+    
+{% endhighlight %}
+{% endtabs %}
+
+## Switch Editor
+
+In switch editor, [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) is loaded, and DataForm [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor supports bool data type property.
```

Comment:
and DataForm switch editor supports bool data type property

Reply:
changed